screen-capture-api/element-capture/index.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Element Capture API demo</title>
+  <link href="index.css" rel="stylesheet">
+  <script defer src="index.js"></script>
+</head>
+
+<body>
+  <h1>Element Capture API example</h1>
+  <p>
+    This example captures the demo container element.
+    Click the Start Capture button to begin.
```

Comment:
Can you mention in this `<p>` what you should see when you click Start Capture? The box on the right broadcast to the box on the left?

Reply:
Sure, I can do that.

Reply:
Done in latest commit.